src/lib/data.json

```diff
@@ -0,0 +1,226 @@
+{
+    "supportedYears": [
+      {
+        "value": "2018",
+        "label": "2018"
+      },
+      {
+        "value": "2019",
+        "label": "2019"
+      },
+      {
+        "value": "2020",
+        "label": "2020"
+      },
+      {
+        "value": "2021",
+        "label": "2021"
+      },
+      {
+        "value": "2022",
+        "label": "2022"
+      }
+    ],
+    
+    "personType": [
+      {
+        "value": "single",
+        "label": "Single"
+      },
+      {
+        "value": "separate",
+        "label": "Separate"
+      },
+      {
+        "value": "together",
+        "label": "Together"
+      },
+      {
+        "value": "head",
+        "label": "Head"
+      }
+    ],
+      "2018": {
+          "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+          "single": [0, 9525, 38700, 82500, 157500, 200000, 500000],
+          "separate": [0, 9525, 38700, 82500, 157500, 200000, 300000],
```

Comment:
Okay, so I know I spelled separate as seperate and that's wrong, but if you correct the spelling, can you please change all instances of seperate into separate? I searched the changes, and there's 8 instances of seperate and 4 instances of seperate. I think this is why you're running into the issue with the calculator breaking. If you do change the name, please also change seperate into separate in helper.js as well or medicare calcultions would get messed up too. Your IDE should have a replace all text in all files option if you do ctrl + shift + f.